metrics/nist_mt/nist_mt.py

```diff
@@ -106,14 +106,14 @@ def _info(self):
 
     def _compute(self, predictions, references, n: int = 5, **tokenize_kwargs):
```

Comment:
I think the issue with the tests is about the signature of compute. Could you try the following:

```suggestion
    def _compute(self, predictions, references, n: int = 5, tokenize_kwargs: Optional[Dict] = None):
```

Reply:
To no avail. It also occurs outside of the testing environment.

```python
import evaluate
nist_mt = evaluate.load("metrics/nist_mt")
hypothesis1 = "It is a guide to action which ensures that the military always obeys the commands of the party"
reference1 = "It is a guide to action that ensures that the military will forever heed Party commands"
reference2 = "It is the guiding principle which guarantees the military forces always being under the command of the Party"
nist_mt.compute(hypothesis1, [reference1, reference2])
```

Error:
```
Traceback (most recent call last):
  File "c:\users\bramv\appdata\local\programs\python\python37\lib\code.py", line 90, in runcode
    exec(code, self.locals)
  File "<input>", line 5, in <module>
TypeError: compute() takes 1 positional argument but 3 were given
```

Reply:
One thing that's off in your example is that your first list "dimension" should the batch of samples, so you are missing a bracket:
```python
import evaluate
nist_mt = evaluate.load("metrics/nist_mt")
hypothesis1 = "It is a guide to action which ensures that the military always obeys the commands of the party"
reference1 = "It is a guide to action that ensures that the military will forever heed Party commands"
reference2 = "It is the guiding principle which guarantees the military forces always being under the command of the Party"
nist_mt.compute([hypothesis1], [[reference1, reference2]])
```